### PR TITLE
[OpenVINO-EP] Add missing dependency libs in Dockerfile

### PR DESCRIPTION
**Pull request by smkarlap**
**Description**: 
- Added missing VAD-M hardware dependency libs into Dockerfile.
- Update documentation with instructions on starting HDDL Daemon.

**Motivation and Context**
- Required to be able to run the MCR Docker container on VAD-M hardware

### CODE REVIEW

File: dockerfiles/Dockerfile.openvino [hunk 1]
```diff
@@ -30,12 +30,13 @@ ENV LANG en_US.UTF-8
...
 RUN apt update && \
     apt -y install git sudo wget locales \
-    zip x11-apps lsb-core cpio libboost-python-dev libpng-dev zlib1g-dev libnuma1 ocl-icd-libopencl1 clinfo libboost-filesystem1.58.0 libboost-thread1.58.0 protobuf-compiler libprotoc-dev libusb-1.0-0-dev autoconf automake libtool && \
+    zip x11-apps lsb-core cpio libboost-python-dev libpng-dev zlib1g-dev libnuma1 ocl-icd-libopencl1 clinfo libboost-filesystem1.58.0 libboost-thread1.58.0 protobuf-compiler libprotoc-dev libusb-1.0-0-dev autoconf automake libtool libudev-dev libjson-c-dev && \
     cd ${MY_ROOT} && \
     git clone --recursive -b $ONNXRUNTIME_BRANCH $ONNXRUNTIME_REPO onnxruntime && \
     cp onnxruntime/docs/Privacy.md /code/Privacy.md && \
     cp onnxruntime/dockerfiles/LICENSE-IMAGE.txt /code/LICENSE-IMAGE.txt && \
     cp onnxruntime/ThirdPartyNotices.txt /code/ThirdPartyNotices.txt && \
+    sed -i "s/sudo//" onnxruntime/dockerfiles/scripts/install_common_deps.sh &&\
```
Comment (jywu-msft) [inline dockerfiles/Dockerfile.openvino:39]: why is this needed?
is it just to get rid of the sudo at https://github.com/microsoft/onnxruntime/blob/9f7d2454463e19f585f3acd75e2ca5b128a88a55/dockerfiles/scripts/install_common_deps.sh#L22 ?
if so, can we modify that script and remove the sudo when downloading cmake? i'm not sure why it's needed anyway.

File: dockerfiles/README.md [hunk 2]
```diff
@@ -157,13 +157,19 @@ Use `docker pull` with any of the images and tags below to pull an image and try
...
 ### OpenVINO on VAD-M Accelerator Version
 
-1. Build the docker image from the DockerFile in this repository.
+1.  Download OpenVINO **Full package** for version **2020.2** for Linux on host machine from [this link](https://software.intel.com/en-us/openvino-toolkit/choose-download) and install it with the help of instructions from [this link](https://docs.openvinotoolkit.org/latest/_docs_install_guides_installing_openvino_linux.html)
+
+2. Install the HDDL drivers on the host machine according to the reference in [here](https://docs.openvinotoolkit.org/latest/_docs_install_guides_installing_openvino_linux_ivad_vpu.html)
+
+3. Build the docker image from the DockerFile in this repository.
      ```
       docker build --rm -t onnxruntime-vadr --build-arg DEVICE=VAD-M_FP16 --network host .
```
Comment (jywu-msft) [inline dockerfiles/README.md:166]: looks like there's some inconsistent naming of the tag here. it says onnxruntime-vadr instead of onnxruntime-hddl

File: dockerfiles/Dockerfile.openvino (hunk 1 above)
Comment (smkarlap) [reply dockerfiles/Dockerfile.openvino:39]: This is mainly needed to be able to validate container images behind company proxy servers and with conda python3 envs. There is a 'sudo pip install numpy' within the install_common_deps.sh, which we would like to use pip3 from the conda env. However, using sudo would cause it to use ubuntu 16.04's python2.7's pip instead. An alternative could be to replace sudo with 'sudo -E'.

Comment (jywu-msft) [reply dockerfiles/Dockerfile.openvino:39]: Where do you see sudo pip install numpy in install_common_deps.sh ? There's no sudo in the latest version on master.

File: dockerfiles/README.md (hunk 2 above)
Comment (smkarlap) [reply dockerfiles/README.md]: Replaced both the outdated HDDL and VADR names with the correct 'VAD-M'.

File: dockerfiles/Dockerfile.openvino (hunk 1 above)
Comment (smkarlap) [reply dockerfiles/Dockerfile.openvino:39]: Ah.. my mistake.. i guess folks were having issues with line 66 (wget) behind proxy.

Comment (jywu-msft) [reply dockerfiles/Dockerfile.openvino:39]: yes, then we should get rid of the sudo in line 22. not sure why it's even there.
there isn't one for the wget in line 14.

Comment (smkarlap) [reply dockerfiles/Dockerfile.openvino]: Sounds good. I have removed the sed replacement line to remove sudo from install_common_deps.sh

Comment (jywu-msft) [reply dockerfiles/Dockerfile.openvino]: Will you make that change to remove the sudo from the wget in install_common_deps.sh ?
otherwise you mentioned you were running into issues?

**Comment by jywu-msft** [comment]
will you also modify install_common_deps.sh to remove the sudo invocation of wget?

**Comment by smkarlap** [comment]
'sudo' removed from install_common_deps.sh
